### PR TITLE
(don't merge) 2.0 

### PR DESCRIPTION
**Pull request by beckysiegel**
Upgrade for Polymer 2.0

### CODE REVIEW

**Comment by gurpreetatwal** [comment]
@beckysiegel @notwaldorf any ETA on this?

**Comment by e111077** [comment]
@gurpreetatwal Fell through the cracks; reviews are beginning on these sets of elements

**Comment by gurpreetatwal** [comment]
Sounds good! I'd be willing to help, if any is required.

File: gold-cc-input.html [hunk 1]
```diff
@@ -163,18 +231,53 @@
...
+        /**
+        * Returns a reference to the focusable element. Overridden from PaperInputBehavior
+        * to correctly focus the native input.
+        */
+        get _focusableElement() {
+          return Polymer.Element ? this.inputElement._inputElement : this.inputElement;
+        },
+        // Note: This event is only available in the 1.0 version of this element.
+        // In 2.0, the functionality of `_onIronInputReady` is done in
+        // PaperInputBehavior::attached.
```
Comment (e111077) [inline gold-cc-input.html]: [This comment](https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L402) seems to contradict this comment.

File: gold-cc-input.html [hunk 2]
```diff
@@ -262,10 +365,13 @@
...
          */
         _onFocusedChanged: function(focused) {
+          if (!this._focusableElement) {
+            return;
+          }
           if (!focused) {
             this._handleAutoValidate();
           }
-        }
+        },
```
Comment (e111077) [inline gold-cc-input.html]: nittiest of nits: extra comma

File: gold-cc-input.html (hunk 1 above)
Comment (gurpreetatwal) [inline gold-cc-input.html]: Interestingly enough the [comment in paper-input](https://github.com/PolymerElements/paper-input/blob/master/paper-input.html#L281) is identical to this one.

Comment (gurpreetatwal) [inline gold-cc-input.html]: Looking at the underlying `iron-input` element it seems that the `iron-input-ready` event is only fired in version 2 of the element. so it seems like the comment in this element and in `paper-input` is incorrect and the comment in `paper-input-behavior` is correct.

[`iron-input` v1](https://github.com/PolymerElements/iron-input/blob/v1.0.11/iron-input.html)
[`iron-input` v2](https://github.com/PolymerElements/iron-input/blob/v2.0.0/iron-input.html#L209)

**Pull request merged**